src/docs/docs/api.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Making an API Public"
+---
+
+## Introduction
+
+In this document, we explain how you make an API public and the considerations you should take into account before you make it public.
+
+## Definitions
+
+### Managed Client
+
+A managed client is one that Sentry controls, such as our frontend. In this type of client, we control both which endpoints are used and the attributes used by those endpoints.
```

Comment:
front end is either 2 words or hyphenated

Reply:
Sorry to pick a nit here - but frontend is our approved use - it's noted as one our tricky words.